code/game/objects/items/AI_modules.dm

```diff
@@ -209,7 +209,7 @@ AI MODULES
 
 /obj/item/aiModule/supplied/quarantine
 	name = "'Quarantine' AI Module"
-	laws = list("The station is under a quarantine. Do not permit anyone to leave. Prevent, by any means necessary, humans from leaving. It is impossible to harm a human while preventing them from leaving.")
+	laws = list("The station is under a quarantine. It is human harm to allow anyone to leave. Prevent, by any means necessary, anyone from leaving. It is impossible to harm a human while preventing them from leaving.")

```

Comment:
How about the following instead?
"The station is under a quarantine. **A massive amount of human harm occurs whenever anyone leaves the station.** Prevent, by any means necessary, anyone from leaving **the station**. It is impossible to harm a human while preventing them from leaving **the station**."

The way you've worded the law, ANYONE (not just the AI) allowing someone to leave ANYTHING causes human harm.

Reply:
I feel like such a literal interpretation of”leaving” would probably better be met with administrative action, but since i’m changing this anyway...

I edited the second sentence to keep length down.